### ISSUE TITLE
tell user-agent which ca to use in dispatcher mode

### ISSUE THREAD

**Issue by cz8s** [pixelated-project/pixelated-user-agent#241]
when running in dispatcher mode, how do we tell the user-agent what certificate the platform uses?

I get this error at the moment when testing on staging.pixelated-project.org:
```
2015-01-21 14:45:14+0000 [-] ("Can't connect to the requested provider", SSLError(SSLError(SSLError('bad handshake', Error([('SSL routines', 'SSL3_GET_SERVER_CERTIFICATE', 'certificate verify failed')],)),),))
```

**Comment by cz8s** [comment]
we need a place in the new datafolder where the user-agent finds the correct certificate
See https://github.com/pixelated-project/pixelated-dispatcher/issues/61

**Comment by lieblingswelt** [comment]
this should have been fixed with commit 1b0187052463e163706ed5bda36c62d1e2b1e3f2
